src/main/java/by/itech/lab/supplier/controller/WarehouseController.java

```diff
@@ -6,17 +6,24 @@
 import org.springframework.data.domain.Page;
 import org.springframework.data.domain.Pageable;
 import org.springframework.data.web.PageableDefault;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.DeleteMapping;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
 import org.springframework.web.bind.annotation.RestController;
 
-import static by.itech.lab.supplier.constant.ApiConstants.URL_ID_PARAMETER;
-import static by.itech.lab.supplier.constant.ApiConstants.URL_WAREHOUSE;
+import javax.validation.Valid;
+
+import static by.itech.lab.supplier.constant.ApiConstants.*;
 
 @RestController
 @AllArgsConstructor
-@RequestMapping(URL_WAREHOUSE)
+@RequestMapping(URL_CUSTOMER + URL_CUSTOMER_ID + URL_WAREHOUSE)
```

Comment:
I think usage of constant here brings more issues than solves. It's hard to read and feels cumbersome.

Reply:
we should talk about it on the meeting